### ISSUE TITLE
Parameter simulation to support multiple parameters

### ISSUE THREAD

**Issue by HuifangYeo** [atoti/atoti#321]
With regards to the [Formula One notebook](https://github.com/atoti/notebooks/blob/master/notebooks/formula-one/main.ipynb) in the notebook gallery, the existing measure simulation involves 2 measures:

```
# Creation of simulation
pointssystem_sim = f1cube.setup_simulation(
    "pointssystem_sim",
    levels=[lvl["positionText"], lvl["resultId"]],
    replace=[m["points.SUM"], m["Best results"]],
    base_scenario="Base",
)
```

This would translate to 2 parameter simulations:

```
result_simulation = f1cube.create_parameter_simulation(
    "Result simulation",
    measure_name="Best results",
    default_value=1000.0,
    levels=[l["resultId"]],
)

points_simulation = f1cube.create_parameter_simulation(
    "Points simulation",
    measure_name="Points parameter",
    levels=[l["positionText"], l["resultId"]],
)
```
They will both have the same number of scenarios:
```
scoring_systems = [
    "Scoring 1950 to 1953",
    "Scoring 1954 to 1957 & 1959",
    "Scoring 1958",
    "Scoring 1960",
    "Scoring 1961 to 1962 & 1966",
    "Scoring 1963 to 1965",
    "Scoring 1967 & 1969 & 1971",
    "Scoring 1968 & 1972",
    "Scoring 1970",
    "Scoring 1973 to 1974",
    "Scoring 1975",
    "Scoring 1976 & 1978",
    "Scoring 1977",
    "Scoring 1979",
    "Scoring 1980",
    "Scoring 1981 to 1990",
    "Scoring 1991 to 2002",
    "Scoring 2003 to 2009",
    "Scoring 2010 to 2020",
]
```
I will have problem trying to create the same visualization for the results where I only display the result for the scenarios with the same name. It would be nice if we could support multiple parameters in a simulation.

**Comment by HuifangYeo** [comment]
![image](https://user-images.githubusercontent.com/35210495/126959141-6572490d-7570-4e53-aa53-05e8dbda8eef.png)

**Comment by bloussou** [comment]
This is available from the `0.6.1` release : https://docs.atoti.io/latest/releases/0.6.1.html#changed
